lib/vanagon/project.rb

```diff
@@ -683,7 +683,7 @@ def publish_yaml_settings(platform)
       end
 
       File.open("#{filepath}.sha1", 'w') do |f|
-        f.write(system("#{platform.shasum} #{filepath}", err: File::NULL))
+        f.write(%x(#{platform.shasum} #{filepath}))
```

Comment:
I misunderstood where this was being executed - This happens locally. I don't want to rely on a `shasum` executable being available this way, though - does vanagon have some idiomatic way of determining the right shasum executable for the local host? If not I could try a few possibilities and error if they all fail...